schemas/user.py

```diff
@@ -96,24 +93,21 @@ def validate_email(cls, v):
                            '[A-Za-z0-9' + specials + ']+(?<!['+ specials + '])@[A-Za-z0-9.-]+\.[A-Za-z]{2,4}$')
         message = False
         if not v:
-            message = "Електронний адрес не може бути порожнім"
+            message = "The email address cannot be empty."
         elif not re.fullmatch(regex, v):
-            message = f"Невірний формат адреси електронної пошти: {v}."
+            message = f"Invalid email address format: {v}."
         if message:
-            raise ValueError(message)
-        
+            raise BackendException(message=message)
```

Comment:
Don't use `BackendException` inside Pydantic schemas validation. it should be handled by another handler: `validation_exception_handler`.